### PR TITLE
Fix tab order in basic auth login screen

### PR DESCRIPTION
**Pull request by erikjv**
Fixes: #11912

### CODE REVIEW

File: src/gui/qml/credentials/BasicAuthCredentials.qml [hunk 1]
```diff
@@ -77,7 +79,7 @@ Credentials {
     Button {
         id: loginButton
         Layout.alignment: Qt.AlignHCenter
-        Layout.preferredWidth: passwordField.width
+        Layout.preferredWidth: userNameField.implicitWidth
```
Comment (TheOneRing) [inline src/gui/qml/credentials/BasicAuthCredentials.qml:82]: Wrong pr?

**Comment by erikjv** [comment]
Title has been updated to reflect that these are 3 commits fixing multiple issues.

File: src/gui/qml/credentials/BasicAuthCredentials.qml (hunk 1 above)
Comment (TheOneRing) [reply src/gui/qml/credentials/BasicAuthCredentials.qml:82]: As we also have 3 issues for that those also need linking

**Review by TheOneRing — DISMISSED** [review]
Lgtm

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]
one pr to fix one issue please

**Comment by erikjv** [comment]
It's now split up

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**